docs/src/pages/components/tabs/tabs.md

```diff
@@ -91,6 +91,29 @@ Tab labels may be either all icons or all text.
 
 {{"demo": "pages/components/tabs/IconLabelTabs.js", "bg": true}}
 
+## Accessibility
+
+(WAI-ARIA: https://www.w3.org/TR/wai-aria-practices/#tabpanel)
+
+This section lists necessary steps to provide necessary information for assistive technology:
```

Comment:
```suggestion
The following steps are needed in order to provide necessary information for assistive technologies:
```
cc @joshwooding

Reply:
So much better. I'm trying to avoid "this"/"that" more and more but this is not that easy :smile: